packages/react-core/src/components/Toolbar/ToolbarGroup.tsx

```diff
@@ -65,9 +75,14 @@ class ToolbarGroupWithRef extends React.Component<ToolbarGroupProps> {
               styles.toolbarGroup,
               variant && styles.modifiers[toCamel(variant) as 'filterGroup' | 'iconButtonGroup' | 'buttonGroup'],
               formatBreakpointMods(visibility, styles, '', getBreakpoint(width)),
-              formatBreakpointMods(alignment, styles, '', getBreakpoint(width)),
```

Comment:
We'll still need this to generate the classes. It allows `align` to be an object and builds out classes like `.pf-m-align-left-on-2xl` to align something to the left at the 2xl breakpoint. Same for `<ToolbarItem>`